### PR TITLE
fix title

### PR DESCRIPTION
**Pull request by stavares843**
### What does this pull request aim to achieve? 💯

fix title

https://github.com/FightPandemics/FightPandemics/issues/442

### CODE REVIEW

File: client/public/manifest.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "short_name": "React App",
+  "short_name": "FightPandemics",
   "name": "Create React App Sample",
```
Comment (anniegiang) [inline client/public/manifest.json:3]: Let's say "FightPandemics" here too

Comment (stavares843) [reply client/public/manifest.json:3]: https://github.com/FightPandemics/FightPandemics/issues/442#issuecomment-632919293

Comment (anniegiang) [reply client/public/manifest.json:3]: Do you think we should merge for now? Or should we wait for more inputs?

Comment (stavares843) [reply client/public/manifest.json:3]: will check if there's any more suggestions

Comment (robinv85) [inline client/public/manifest.json:3]: and "name" ?

Comment (stavares843) [reply client/public/manifest.json:3]: https://github.com/FightPandemics/FightPandemics/issues/442#issuecomment-632919293

Comment (stavares843) [reply client/public/manifest.json]: updated

Comment (stavares843) [reply client/public/manifest.json]: updated

**Pull request merged**